packages/cli/src/bin.ts

```diff
@@ -54,9 +54,11 @@ process.removeAllListeners('warning')
 const debug = Debug('prisma:cli')
 process.on('uncaughtException', (e) => {
   debug(e)
+  throw e
 })
 process.on('unhandledRejection', (e) => {
   debug(e)
+  throw e
```

Comment:
This is unrelated to the referenced issues and can be extracted to a separate PR. These handlers could also lead to the CLI process hanging in some cases as they silently swallowed and ignored any unhandled exceptions (for me the problem occurred when I had a `ReferenceError` in an `exit` event handler in `GeneratorProcess`).

We can also just remove these event handlers if we don't need to log these errors to the debug log specifically.

`git blame` traces all the way back to https://github.com/prisma/prisma/commit/8700714177ada7a8046919a74daab1a9ba567e70, and this code doesn't seem to be present in Prisma 1, so hard to tell what it could have been added for.